sound/soc/soc-jack.c

```diff
@@ -79,6 +79,44 @@ int snd_soc_card_jack_new(struct snd_soc_card *card, const char *id, int type,
 }
 EXPORT_SYMBOL_GPL(snd_soc_card_jack_new);
 
+/**
+ * snd_soc_card_jack_new_ctl - Create a new jack with a kcontrol
```

Comment:
Hmm, this took a while to understand the different variants. Maybe name this snd_soc_card_jack_new_kctl() (like in jack.h snd_jack_add_new_kctl()) ?

Reply:
Right. I will change to xxx_kctl.

Reply:
Yes please, without knowing what problem it fixes or what feature it improves it is hard to understand why we need this change